examples/http-response-interceptors/http_response_interceptors.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+
+// Header name to be set to the response in the response interceptor.
+final string interceptor_header = "responseHeader";
```

Comment:
Do we still need them ?

Reply:
This variable is just there to set a header to the response inside a response interceptor. That is the only indication of  the response interceptor execution.